src/test/java/io/odh/test/e2e/upgrade/OlmUpgradeST.java

```diff
@@ -99,14 +99,15 @@ void testUpgradeOlm() throws IOException, InterruptedException {
 
         LOGGER.info("Upgrade to next available version in OLM catalog");
         // Approve upgrade to newer version
-        KubeUtils.waitForInstallPlan(olmInstall.getNamespace(), olmInstall.getCsvName());
-        ip = ResourceManager.getKubeClient().getNonApprovedInstallPlan(olmInstall.getNamespace(), olmInstall.getCsvName());
-        ResourceManager.getKubeClient().approveInstallPlan(olmInstall.getNamespace(), ip.getMetadata().getName());
+        KubernetesUtils.waitForInstallPlan(olmInstall.getNamespace(), olmInstall.getCsvName());
+        ip = io.skodjob.testframe.utils.KubeUtils.getNonApprovedInstallPlan(olmInstall.getNamespace(), olmInstall.getCsvName());
+        io.skodjob.testframe.utils.KubeUtils.approveInstallPlan(olmInstall.getNamespace(), ip.getMetadata().getName());
```

Comment:
Import `io.skodjob.testframe.utils.KubeUtils` in this file?

Reply:
It is draft...